source/stylesheets/layout.sass

```diff
@@ -218,6 +217,7 @@ header
       text-decoration: none
 
 .store-header
+  color: var(--shop-name-text-color)
   font-size: 42px
   letter-spacing: 4px
```

Comment:
not sure this letter-spacing value still makes sense here (along with all of the others sprinkled throughout)